infra/app/service/main.tf

```diff
@@ -35,6 +35,8 @@ locals {
 
   service_name = "${local.prefix}${module.app_config.app_name}-${var.environment_name}"
 
+  isTemporary = startswith(terraform.workspace, "t-") || startswith(module.project_config.project_name, "plt-tst-act-")
```

Comment:
nit, terraform naming convention is snake_case rather than camelCase like it is in javascript